# Translations template for ironic.
# Copyright (C) 2014 ORGANIZATION
# This file is distributed under the same license as the ironic project.
#
# Translators:
# Andi Chandler <andi@gowling.com>, 2014
msgid ""
msgstr ""
"Project-Id-Version: Ironic\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2014-09-02 06:14+0000\n"
"PO-Revision-Date: 2014-08-29 19:48+0000\n"
"Last-Translator: openstackjenkins <jenkins@openstack.org>\n"
"Language-Team: English (United Kingdom) (http://www.transifex.com/projects/p/"
"ironic/language/en_GB/)\n"
"Language: en_GB\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 1.3\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: ironic/conductor/manager.py:403
#, python-format
msgid ""
"No free conductor workers available to perform an action on node %(node)s, "
"setting node's provision_state back to %(prov_state)s and "
"target_provision_state to %(tgt_prov_state)s."
msgstr ""
"No free conductor workers available to perform an action on node %(node)s, "
"setting node's provision_state back to %(prov_state)s and "
"target_provision_state to %(tgt_prov_state)s."

#: ironic/conductor/manager.py:1129
#, python-format
msgid ""
"get_sensors_data is not implemented for driver %(driver)s, node_uuid is "
"%(node)s"
msgstr ""
"get_sensors_data is not implemented for driver %(driver)s, node_uuid is "
"%(node)s"

#: ironic/conductor/manager.py:1133
#, python-format
msgid ""
"During get_sensors_data, could not parse sensor data for node %(node)s. "
"Error: %(err)s."
msgstr ""
"During get_sensors_data, could not parse sensor data for node %(node)s. "
"Error: %(err)s."

#: ironic/conductor/manager.py:1137
#, python-format
msgid ""
"During get_sensors_data, could not get sensor data for node %(node)s. Error: "
"%(err)s."
msgstr ""
"During get_sensors_data, could not get sensor data for node %(node)s. Error: "
"%(err)s."

#: ironic/conductor/manager.py:1141
#, python-format
msgid ""
"During send_sensor_data, node %(node)s was not found and presumed deleted by "
"another process."
msgstr ""
"During send_sensor_data, node %(node)s was not found and presumed deleted by "
"another process."

#: ironic/conductor/manager.py:1145
#, python-format
msgid "Failed to get sensor data for node %(node)s. Error: %(error)s"
msgstr "Failed to get sensor data for node %(node)s. Error: %(error)s"

#: ironic/conductor/task_manager.py:277
#, python-format
msgid "Task's on_error hook failed to call %(method)s on node %(node)s"
msgstr "Task's on_error hook failed to call %(method)s on node %(node)s"

#: ironic/dhcp/neutron.py:145
#, python-format
msgid ""
"No VIFs found for node %(node)s when attempting to update DHCP BOOT options."
msgstr ""

#: ironic/dhcp/neutron.py:163
#, python-format
msgid ""
"Some errors were encountered when updating the DHCP BOOT options for node "
"%(node)s on the following ports: %(ports)s."
msgstr ""

#: ironic/drivers/modules/agent.py:536
#, python-format
msgid "Malformed MAC: %s"
msgstr "Malformed MAC: %s"

#: ironic/drivers/modules/agent.py:575
#, python-format
msgid "MAC address %s not found in database"
msgstr "MAC address %s not found in database"

#: ironic/drivers/modules/console_utils.py:179
#, python-format
msgid ""
"Failed to kill the old console process before starting a new shellinabox "
"console for node %(node)s. Reason: %(err)s"
msgstr ""

#: ironic/drivers/modules/iboot.py:115
#, python-format
msgid ""
"Cannot get power status for node '%(node)s' at relay '%(relay)s'. iBoot "
"get_relays() returned '%(resp)s'."
msgstr ""

#: ironic/drivers/modules/ipmitool.py:704
#, python-format
msgid ""
"IPMI set boot device failed for node %(node)s when executing \"ipmitool "
"%(cmd)s\". Error: %(error)s"
msgstr ""
"IPMI set boot device failed for node %(node)s when executing \"ipmitool "
"%(cmd)s\". Error: %(error)s"

#: ironic/drivers/modules/ipmitool.py:735
#, python-format
msgid ""
"IPMI get boot device failed for node %(node)s when executing \"ipmitool "
"%(cmd)s\". Error: %(error)s"
msgstr ""
"IPMI get boot device failed for node %(node)s when executing \"ipmitool "
"%(cmd)s\". Error: %(error)s"

#: ironic/drivers/modules/ssh.py:677
#, python-format
msgid ""
"Failed to get boot device for node %(node)s, virt_type %(vtype)s does not "
"support this operation"
msgstr ""
"Failed to get boot device for node %(node)s, virt_type %(vtype)s does not "
"support this operation"

#: ironic/nova/virt/ironic/driver.py:98
#, python-format
msgid "Power state %s not found."
msgstr "Power state %s not found."

#: ironic/nova/virt/ironic/driver.py:212
#, python-format
msgid "Ignoring malformed capability '%s'. Format should be 'key:val'."
msgstr "Ignoring malformed capability '%s'. Format should be 'key:val'."

#: ironic/nova/virt/ironic/driver.py:503
#, python-format
msgid "Warning, memory usage is 0 for %(instance)s on baremetal node %(node)s."
msgstr ""

#: ironic/nova/virt/ironic/driver.py:510
#, python-format
msgid ""
"Warning, number of cpus is 0 for %(instance)s on baremetal node %(node)s."
msgstr ""

#: ironic/nova/virt/ironic/driver.py:695
#, python-format
msgid "Destroy called on non-existing instance %s."
msgstr "Destroy called on non-existing instance %s."

#: ironic/openstack/common/loopingcall.py:87
#, python-format
msgid "task %(func_name)s run outlasted interval by %(delay).2f sec"
msgstr "task %(func_name)s run outlasted interval by %(delay).2f sec"

#: ironic/openstack/common/network_utils.py:149
msgid "tcp_keepidle not available on your system"
msgstr "tcp_keepidle not available on your system"

#: ironic/openstack/common/network_utils.py:156
msgid "tcp_keepintvl not available on your system"
msgstr "tcp_keepintvl not available on your system"

#: ironic/openstack/common/network_utils.py:163
msgid "tcp_keepknt not available on your system"
msgstr "tcp_keepknt not available on your system"

#: ironic/openstack/common/service.py:364
#, python-format
msgid "pid %d not in child list"
msgstr "pid %d not in child list"
